projects/step-core/src/lib/modules/basics/shared/is-small-screen.token.ts

```diff
@@ -0,0 +1,16 @@
+import { inject, InjectionToken } from '@angular/core';
+import { SCREEN_WIDTH } from './screen-width.token';
+import { distinctUntilChanged, map, Observable } from 'rxjs';
+
+const SMALL_SCREEN_WIDTH = 850;
```

Comment:
hmm shouldn't we use the values used by bootstrap / tailwind

<img width="550" alt="image" src="https://user-images.githubusercontent.com/8954749/232807681-0153741c-5ec3-4524-97c2-abc99092b4ca.png">

I think it would be better to use values that are kind of community agreed upon, unless we have a specific case to introduce ones of our own

Reply:
@bojidaryovchev In comparison here is the bootstrap endpoints
![изображение](https://user-images.githubusercontent.com/16287112/233353616-19e0850f-6949-4735-a3cf-b59e536344f9.png)
As you see, it uses absolutely different size breakpoints in comparison with tailwind.
Both of these sets of breakpoints are common in terms of usage of  the concrete style library.
But as we don't use any of these libraries, I think we can define our own dimensions, which are suitable for our needs. 
It's a good idea to standardize it in terms STEP design guide - but I think it should be done under another issue that will be related to global mobile version (not minimal viable solution)

Reply:
Yes for this ticket it is sufficient like this. I agree that we should think about breakpoints (and maybe use those defined by other frameworks) when we do the real mobile version of step.